requirements_dev.txt

```diff
@@ -1,12 +0,0 @@
-pip==9.0.1
```

Comment:
We need this file for development setup, extra things that you need to install for developing. Hence, the `_dev` suffix. This includes linters and testing utilities. So, I'd recommend to not delete this file.

Reply:
I really wish we could define such things in `test_requires` in `setup.py`, since both linting and testing are Quality Assurance aspects of writing the code, which inherently makes those *test requirements*. I can see however that using `test_requirements` again makes using wheel-only test dependencies problematic.

I think a nice compromise would be to have `test_requires` in `setup.py` use a `requirements_test.txt` file so that they are both installable with `pip` and usable for `setup.py`